src/pages/home/home.styled.ts

```diff
@@ -0,0 +1,16 @@
+export const CONTENT_CLASSES = {
+  CONTENT_1: {
```

Comment:
Can you name these what they show? ie `HOST_BENEFITS` etc

Reply:
sorry that was a placeholder name, i changed it to `FEATURES` based on the sketch file but i can change it to `BENEFITS` if that's what we want

Reply:
Features work too